easybuild/easyconfigs/b/binutils/binutils-2.28-GCCcore-6.4.0.eb

```diff
@@ -1,21 +1,25 @@
+easyblock = 'EB_binutils'
```

Comment:
not needed (although again, I understand the motivation); this is basically duplicate information